### PR TITLE
Increase error reporting on iiod and network backend

### PR DESCRIPTION
**Pull request by rgetz**
This was taken from Lar's old branch "stresstest"
https://github.com/analogdevicesinc/libiio/commit/dc337057506507c35ab7185af810048eb2e69259

and adds a few more error output when things go wrong.

Signed-off-by: Lars-Peter Clausen <lars@metafoo.de>
Signed-off-by: Robin Getz <robin.getz@analog.com>

### CODE REVIEW

**Comment by rgetz** [comment]
This is got problem on Windows - don’t merge yet.

**Comment by rgetz** [comment]
Ok - this cleans things up, and gets ride of all the warnings.

OK to merge when someone has time.

File: network.c [hunk 1]
```diff
@@ -735,8 +735,10 @@ static int network_open(const struct iio_device *dev,
 		goto out_mutex_unlock;
 
 	ret = create_socket(pdata->addrinfo, DEFAULT_TIMEOUT_MS);
-	if (ret < 0)
+	if (ret < 0) {
+		ERROR("Create socket: %d\n", ret);
```
Comment (dNechita) [inline network.c]: Just a minor thing: I saw that "Create socket" and "Open unlocked" have lower case but the rest of messages are all uper case.

File: network.c [hunk 2]
```diff
@@ -745,8 +747,10 @@ static int network_open(const struct iio_device *dev,
 
 	ret = iiod_client_open_unlocked(pdata->iiod_client,
 			&ppdata->io_ctx, dev, samples_count, cyclic);
-	if (ret < 0)
+	if (ret < 0) {
+		ERROR("Opend unlocked: %d\n", ret);
```
Comment (dNechita) [inline network.c]: opend typo

**Pull request merged**